core/src/main/java/org/kohsuke/stapler/jsr269/QueryParameterAnnotationProcessor.java

```diff
@@ -69,9 +69,9 @@ private void write(ExecutableElement m) throws IOException {
         }
 
         TypeElement t = (TypeElement) m.getEnclosingElement();
-        FileObject f = createResource(
-                t.getQualifiedName().toString().replace('.', '/') + "/" + m.getSimpleName() + ".stapler");
-        notice("Generating " + f, m);
+        String name = t.getQualifiedName().toString().replace('.', '/') + "/" + m.getSimpleName() + ".stapler";
+        FileObject f = createResource(name);
+        notice("Generating " + name, m);
```

Comment:
This was logging an object without a `toString()` method, which made it impossible to expect a specific value in the tests because the default `Object#toString` prints a memory address. By printing the name instead, we have a reliable thing to check for in tests, as well as prettier debug output.